src/OrleansRuntime/Silo/Silo.cs

```diff
@@ -222,7 +238,6 @@ internal Silo(string name, SiloType siloType, ClusterConfiguration config, ILoca
 
             // Configure DI using Startup type
             bool usingCustomServiceProvider;
```

Comment:
Is this variable still needed here? I believe it is not used anywhere after this change.